mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -686,11 +686,13 @@ public static Response listContents(UriInfo info, Application app, Request reque
                         String result = listDerivateContentAsJson(derObj, path, depth, info, app);
                         return response(result, "application/json", lastModified);
                     }
+                    // fallthrough
```

Comment:
The if can be removed It checks the the same condition as the case expression -> no fallthrough